src/DropdownButton.js

```diff
@@ -15,7 +15,10 @@ const DropdownButton = OutlineButton.extend.attrs({
   height: 100%;
   min-height: 44px;
   cursor: ${props => props.cursor};
-  border-color: ${theme('colors.borderGray')};
+  border-top-color: ${theme('colors.borderGray')};
```

Comment:
why not `${props => props.theme.colors.borderGray}` ?

Reply:
@hakimelek , good catch. Updated. There's no special use case here to use the `theme(...)` utility. More correct to use the color(s) provided by client's theme provider.

Reply:
The `styledSystem.theme` utility is an existential getter, which won't throw if `theme.colors` is undefined; it can also accept a fallback value – that's the only difference between the syntax change here

Reply:
Yes, sorry I was not aware of what `theme` from `styled-system` is doing! It might be better to adopt it as an alternative pattern to use the `theme` props

Reply:
The fallback value of the `systemStyle.theme` utility sounds interesting. Gonna try out the theme utility with a fallback value in the next design system feature, if there's a use case of theme. Thanks for the educational information, @jxnblk . +1